src/templates/partials/course-list-table.twig

```diff
@@ -0,0 +1,104 @@
+{% set filter_active = false %}
+{% set minor_badge = '<span class="badge badge--primary ml-2">Major</span>' %}
+{% set major_badge = '<span class="badge badge--primary ml-2 ml-md-7 ml-lg-8">Minor</span>' %}
+
+{% embed 'partials/sidebar-layout.twig' %}
+	{% block nav %}
+		{% include 'partials/event-search.twig' with {
+      title: 'Search Courses'
+    } %}
+		{% include 'partials/event-filters.twig' %}
+	{% endblock %}
+	{% block content %}
+		{% if filter_active %}
+			{% include 'partials/active-filters.twig' %}
+		{% endif %}
+
+		{% if no_results %}
+			<p class="lead">
+				There are no events on this date.
+			</p>
+		{% endif %}
+
+		<p style="padding-bottom:48px">Middlebury College's liberal arts curriculum will challenge you for the rest of your life. Our
+			<a href="#">undergraduate</a>
+			and
+			<a href="#">graduate programs</a>
+			prepare students to meet the challenges of responsible citizenship in a complex, changing world.<br><br>Our curriculum is designed to ensure that each student's education includes breadth of experience across many fields and disciplines, as well as in-depth study in one area defined by the
+			<a href="http://www.middlebury.edu/academics/options">major</a>. Breadth is achieved through a set of
+			<a href="http://www.middlebury.edu/about/handbook/ug-college-policies/ug-policies/academics/degree-requires">distribution requirements</a>
+			that encompass seven academic categories and four courses in different cultures and civilizations. An
+			<a href="#">emphasis on writing</a>
+			in all disciplines sharpens students' capacity for critical thinking and expression.
+			<br><br>An array of interdisciplinary programs provide opportunities to synthesize and connect what you learn in many different departments. Intensive study in the major may culminate in a senior thesis, independent project, or artistic production, working closely with professors in laboratories, studios and classrooms to collaborate on research and creative projects. You'll have a broad range of choices, and a great deal of freedom to set your own academic path.</p>
+
+		<table class="table--course-list">
+			<thead class="sr-only">
+				<th>
+					Area of Study
+				</th>
+				<th>
+					Major
+				</th>
+				<th>
+					Minor
+				</th>
+			</thead>
+			<thead>
```

Comment:
I'm confused about the second table header here. The WAVE accessibility checker throws an error for empty <th> cells. Can this be reverted to the previous setup where we had a single <thead> with values in each <th>?

Reply:
Hey Ian, that was a workaround I put it in because the header row was not displaying correctly on Safari with the other two cells being sr-only. I was looking into the accessibility of it and other ways of making this work. I'm still working on making it work with one header row. I'll push a fix soon for this.